Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: all clean compile stop
+
+all: stop clean compile
+
+# deep clean
+clean:
+	./gradlew clean
+	rm -rf distributions
+
+# build and run tests
+compile:
+	./gradlew build
+	xcodebuild build -workspace ./sentry-samples/kmp-app/iosApp/iosApp.xcworkspace -scheme iosApp CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
```

Comment:
So `build` does not trigger the iOS build automatically?

Reply:
I'll double check, I'm currently investigating it right now because it succeeded on ubuntu as well